pmemcheck/pmc_main.c

```diff
@@ -296,18 +302,193 @@ is_pmem_access(Addr addr, SizeT size)
 }
 
 /**
+* \brief State to string change for information purposes.
+*/
+static const char *
+store_state_to_string(enum store_state state)
+{
+    switch (state) {
+        case STST_CLEAN:
+            return "CLEAN";
+        case STST_DIRTY:
+            return "DIRTY";
+        case STST_FLUSHED:
+            return "FLUSHED";
+        case STST_FENCED:
+            return "FENCED";
+        case STST_COMMITTED:
+            return "COMMITTED";
+        default:
+            return NULL;
+    }
+}
+
+/**
+ * \brief Prints registered flush errors.
```

Comment:
I would expect better explanation what *error* means in this case.  Honestly, it's not really an error, but performance issue.